cmd/generate/generate.go

```diff
@@ -159,15 +159,15 @@ func Validate(o *GenerateOptions) error {
 			o.format, strings.Join(validFormats, ", "))
 	}
 
-	if o.customEntriesPath == "" {
-		return nil
+	if o.customEntriesPath != "" && o.customEntriesFormat == "" {
```

Comment:
let's wrap everything related to customEntries in a validateCustomEntries func, and return nil first thing when they are both empty.